docs/design/downloads.md

```diff
@@ -0,0 +1,100 @@
+# Edge Image Builder Downloads
+
+## EIB Builder Image Container Downloads
+
+These packages are necessary for building the EIB binary.
+```
+gpgme-devel
+device-mapper-devel
+libbtrfs-devel
+```
+
+## EIB Deliverable Image Container Downloads
+
+These packages are used by EIB to build the RTD image. Some of these package binaries may be copied to the RTD image.
+
+Repository
+```
+https://download.opensuse.org/repositories/isv:SUSE:Edge:EdgeImageBuilder/SLE-15-SP5/isv:SUSE:Edge:EdgeImageBuilder.repo
+```
+Packages
+```
+xorriso
+squashfs
+libguestfs
+kernel-default
+e2fsprogs
+parted
+gptfdisk
+btrfsprogs
+guestfs-tools
+lvm2
+podman
+createrepo_c
+helm
+hauler
+nm-configurator 
+```
+
+## EIB Deliverable Image Programmatic Downloads
+
+These artifacts are programmatically downloaded by EIB during build time. Some of these artifacts may be copied to the RTD image.
+
+### Elemental
+Source
+```
+https://download.opensuse.org/repositories/isv:/Rancher:/Elemental:/Staging/standard/
```

Comment:
We're missing a few packages from the repo, but in the coming days the new repos will be:

https://updates.suse.com/SUSE/Products/ElementalTeal/5.4/x86_64/product

Reply:
We can update it as needed in future PRs but will land this for now 👍